nixos/modules/services/misc/gitlab.nix

```diff
@@ -120,6 +120,14 @@ let
           port = 3807;
         };
       };
+      registry = {
+        enabled = cfg.registry.enable;
+        host = cfg.registry.host;
+        port = cfg.registry.port;
+        key = cfg.registry.keyFile;
+        api_url = "http://localhost:5000/";
```

Comment:
Except for `cfg.registry.enable`, we should just direct users to use the `services.dockerRegistry.*` options, and read back from them when configuring gitlab.

This would allow users to customize the registry even further when needed, and we wouldn't have to duplicate defining configuration options here and in there.